pkgs/by-name/sm/smctemp/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  darwin,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "smctemp";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "narugit";
+    repo = pname;
+    rev = version;
+    hash = "sha256-opkE4XYe2x+4TwZ7AZ001NDJ0eC4JXcHwMKzkQA1E0w=";
+  };
+
+  buildInputs = lib.optionals stdenv.isDarwin [
+    darwin.IOKit
+  ];
+
+  patchPhase = ''
+    substituteInPlace Makefile \
+      --replace 'CXX := g++' 'CXX := c++' \
+      --replace 'CXXFLAGS := -Wall -std=c++17 -g -framework IOKit' \
+                'CXXFLAGS := -Wall -std=c++17 -g -framework IOKit -Wno-deprecated-declarations -Wno-format-security' \
```

Comment:
This seems something we only want on darwin platforms

Reply:
Yeah, though the package itself is for Darwin only.

Reply:
This is absolutely true 😅

Reply:
I'm sorry then, I probably misunderstood you. I thought you implied to add a platform if or something.

Reply:
no, no it's my bad